analytics/report/src/report_generation/report_generator.py

```diff
@@ -0,0 +1,84 @@
+print("Starting script...")
+# import pandas as pd
+import os
+from jinja2 import Environment, FileSystemLoader
+
+# Sample data
+data = {
+    "listing_id": [1, 2, 3, 4, 5, 6, 7],
+    "address": ["11 ABC street.", "22 ABC street.", "33 ABC street.", "44  ABC street.", "55  ABC street.", "66  ABC street.", "77  ABC street."],
+    "policy_A": [True, False, True, False, True, False, True]
+}
+
+
+########################################################################
+########################################################################
+# USING JINJA2 TO HTML
+########################################################################
+########################################################################
+
+
+# Get the absolute path to the directory containing report_generator.py
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+# Use this path to set the correct path to the template/ directory
+template_dir = os.path.join(dir_path, 'template')
+
+
```

Comment:
these should be encapsulated into its own class and it should invoke the policy class. i think alja had some pseudo code in the readme whcih demonstrates how the report generation should be done. @alchuu00